x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/rules_list/components/rules_list_table.tsx

```diff
@@ -169,6 +169,7 @@ export function convertRulesToTableItems(opts: ConvertRulesToTableItemsOpts): Ru
         (canExecuteActions || (!canExecuteActions && !rule.actions.length)),
       enabledInLicense: !!ruleTypeIndex.get(rule.ruleTypeId)?.enabledInLicense,
       showIntervalWarning: parseDuration(rule.schedule.interval) < minimumDuration,
+      isInternallyManaged: ruleTypeIndex.get(rule.ruleTypeId)!.isInternallyManaged,
```

Comment:
at this point, rules with no valid rule type are already filtered out. That's why I feel confident using the `!`